pkg/kubelet/kubelet.go

```diff
@@ -1554,6 +1555,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
 			glog.V(4).Infof("Periodic sync")
+		case <-syncStatusTicker:
+			glog.V(4).Infof("Periodic status sync")
+			kl.syncStatus()
+			continue
```

Comment:
My vote is to just have a separate Go-routine that syncsStatus() on its own interval. The code should looks simpler (minus having more Go-routines to reason about, but they're well scoped). I don't think we need to place those updates in this sync loop alone.

Also note that the default sync interval is 10s, which would make the syncStatus happen every 30s, is that often enough?